### PR TITLE
docs: split simple and more complex getting started guides

### PR DESCRIPTION
**Pull request by steverfrancis**
# Pull Request

## Acceptance

Please use the following checklist:

- [ ] you linked an issue (if applicable)
- [ ] you included tests (if applicable)
- [ ] you ran conformance (`make conformance`)
- [ ] you formatted your code (`make fmt`)
- [ ] you linted your code (`make lint`)
- [ ] you generated documentation (`make docs`)
- [ ] you ran unit-tests (`make unit-tests`)

> See `make help` for a description of the available targets.

### CODE REVIEW

File: website/content/v1.5/introduction/prodnotes.md [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+> Note: `@cni.patch` means that the patch is read from a file named `cni.patch`.
+
+#### Machine Configs as Templates
+
+Individual machines may need different settings: for instance, each may have a
+different [static IP address]({{< relref "../advanced/advanced-networking/#static-addressing" >}}).
+
+When different files are needed for machines of the same type, there are two supported flows:
+
+1. Use the `talosctl gen config` command to generate a template, and then patch
```
Comment (frezbo) [inline website/content/v1.5/introduction/prodnotes.md:178]: could be worth mentioning `talosctl apply-config --patch` also works

**Review by smira — COMMENTED** [review]
I like the idea of splitting the guides, it needs some cleanup for linting issues, and probably adding some more links where appropriate.

I can clean it up

File: website/content/v1.5/introduction/prodnotes.md (hunk 1 above)
Comment (smira) [reply website/content/v1.5/introduction/prodnotes.md:178]: we should probably try not to mention every option on other hand?

**Review by smira — APPROVED** [review]
did some cleanups, but mostly linter issues

**Review by frezbo — APPROVED** [review]

**Pull request merged**